src/xbps/index.md

```diff
@@ -30,10 +30,14 @@ Most general package management is done with the following commands:
 Most questions can be answered by consulting the man pages for these tools,
 together with the [xbps.d(5)](https://man.voidlinux.org/xbps.d.5) man page.
 
-To learn how to build packages from source, refer to [the README for the
+To learn how to build packages from source, refer to [the README of the
 void-packages
 repository](https://github.com/void-linux/void-packages/blob/master/README.md).
 
+To learn how to create new packages, refer to [the Manual of the
```

Comment:
This doesn't really explain how the `README` and the manual differ.

Reply:
I disagree. Building packages and creating packages seems quite different to me.

Reply:
What do you think about: "To learn how to create new packages templates [...]"?

Reply:
*package templates

Reply:
I changed it to "package templates" now, also ran `vmdfmt` on it. It should pass the test now...